app/views/customers/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @customer.name %></h1>
+<ul>
+  <% @customer.items.each do |item| %>
+    <li><%= "#{item.name} - $#{item.price} (From: #{item.supermarket.name})" %></li>
```

Comment:
This `<li>` element is where you can add a css id that will be unique for each item, then we can do more targeted testing with within blocks.

Reply:
Good job accessing all the items for this customer.